packages/actor-http-cache/README.md

```diff
@@ -0,0 +1,38 @@
+# Comunica Cache Http Actor
+
+[![npm version](https://badge.fury.io/js/%40comunica%2Factor-http-cache.svg)](https://www.npmjs.com/package/@comunica/actor-http-cache)
+
+A comunica Cache Http Actor.
+
+This module is part of the [Comunica framework](https://github.com/comunica/comunica),
+and should only be used by [developers that want to build their own query engine](https://comunica.dev/docs/modify/).
+
+[Click here if you just want to query with Comunica](https://comunica.dev/docs/query/).
+
+## Install
+
+```bash
+$ yarn add @comunica/actor-http-cache
+```
+
+## Configure
+
+After installing, this package can be added to your engine's configuration as follows:
+```text
+{
+  "@context": [
+    ...
+    "https://linkedsoftwaredependencies.org/bundles/npm/@comunica/actor-http-cache/^1.0.0/components/context.jsonld"  
+  ],
+  "actors": [
+    ...
+    {
+      "@id": "urn:comunica:default:http/actors#cache",
+      "@type": "ActorHttpCache",
+      "cacheStorage": { "@id": "urn:comunica:default:http/cache-storage#main" },
+      "mediatorHttpInvalidate": { "@id": "urn:comunica:default:http-invalidate/mediators#main" },
+      "mediatorHttp": { "@id": "urn:comunica:default:http/mediators#main" }
+    },
+  ]
+}
+```
```

Comment:
The convention is to explicitly list config parameters in the readme, like here: https://github.com/comunica/comunica/tree/master/packages/actor-http-proxy
Could we add them here as well?